src/data/actions/portalConfiguration.js

```diff
@@ -34,7 +37,7 @@ const fetchPortalConfiguration = slug => (
     dispatch(fetchPortalConfigurationRequest());
     return LmsApiService.fetchEnterpriseBySlug(slug)
       .then((response) => {
-        dispatch(fetchPortalConfigurationSuccess(response.data));
+        dispatch(fetchPortalConfigurationSuccess(response));
```

Comment:
[context] We want to all of `response` now that it contains both `data` (i.e., the actively viewed enterprise customer metadata) and `enterpriseFeatures` (i.e., the Waffled-based feature flags).